src/signup1.html

```diff
@@ -0,0 +1,49 @@
+@@include('header.htm', { "title": "Sleek - Admin Dashboard Template", "className": "header-fixed sidebar-fixed sidebar-dark header-light" })
```

Comment:
Can you please explain why this file exist?

Reply:
That was unnecessary file. I have deeded that file